src/app/private/scripts/controllers/announcement/resendAnnouncementController.js

```diff
@@ -0,0 +1,420 @@
+'use strict'
```

Comment:
this controller is duplicating code in the sendAnnouncementController. please remove all duplicated code.